libs/platform/src/lib/search-field/search-field.component.ts

```diff
@@ -142,6 +142,20 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
         this._showCategoryDropdown = Array.isArray(value) && value.length > 0;
     }
 
+    /** Current category, value should be present in categories array */
+    @Input()
+    set currentCategory(value: ValueLabelItem) {
+        if (
+            this._showCategoryDropdown &&
```

Comment:
Can we simplify it with includes method?

Reply:
Good catch, thank you!